functional-tests/src/test/groovy/data/data_deletion.sql

```diff
@@ -12,26 +16,49 @@ BEGIN
     );
     DELETE FROM permit_amendment
     WHERE permit_id = ANY (
-    SELECT permit_id FROM permit WHERE create_user LIKE IDIR_USER);
+        SELECT permit_id FROM permit WHERE create_user LIKE IDIR_USER
+    );
     DELETE FROM permit WHERE create_user LIKE IDIR_USER;
     DELETE FROM party_address_xref
     WHERE party_guid = ANY (
-    SELECT party_guid
-    FROM party
-    WHERE create_user LIKE IDIR_USER);
+        SELECT party_guid
+        FROM party
+        WHERE create_user LIKE IDIR_USER
+    );
     DELETE FROM party WHERE create_user LIKE IDIR_USER;
     DELETE FROM mine_location WHERE create_user LIKE IDIR_USER;
     DELETE FROM mine_status_xref WHERE create_user LIKE IDIR_USER;
     DELETE FROM mineral_tenure_xref WHERE create_user LIKE IDIR_USER;
     DELETE FROM mine_status WHERE create_user LIKE IDIR_USER;
     DELETE FROM mine_tailings_storage_facility WHERE create_user LIKE IDIR_USER;
-    DELETE FROM mine_document WHERE create_user LIKE IDIR_USER;
     DELETE FROM mine_expected_document_xref
     WHERE exp_document_guid = ANY(
-    SELECT exp_document_guid
-    FROM mine_expected_document
-    WHERE create_user LIKE IDIR_USER);
+        SELECT exp_document_guid
+        FROM mine_expected_document
```

Comment:
Not sure what the goal is of this `WHERE`. It looks like this will just delete all records from `mine_expected_document_xref`? The original condition was limited to documents created by the `IDIR_USER`

Reply:
good catch O_o

Reply:
fixed